.github/workflows/schedule-last-call-date-end.yml

```diff
@@ -0,0 +1,162 @@
+name: Schedule Status Update Based on Last Call Time
+
+permissions:
+  pull-requests: write
+  contents: write
+
+on:
+  pull_request:
+    types: [closed]
+    branches:
+      - main
+
+jobs:
+  check-merged-file:
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out the code
+        uses: actions/checkout@v2
```

Comment:
Actions should be using commit strings as versions:

`uses: actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744 # v3.6.0`